pkg/upgrade/upgrade.go

```diff
@@ -25,72 +25,44 @@ import (
 	"fmt"
 
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
+	"github.com/cockroachdb/cockroach/pkg/upgrade/upgradebase"
 )
 
-// Upgrade defines a program to be executed once every node in the cluster is
```

Comment:
This seems backwards. Let's leave this in `upgrade` and move the dependencies and structs to a subpackage. It's not used externally.